client/src/components/Header/Header.tsx

```diff
@@ -73,29 +75,31 @@ const HeaderWrapper = styled.div<{
 `;
 
 const Header = () => {
-  const [show, setShow] = useState(true);
-  const [lastScrollY, setLastScrollY] = useState(0);
-
-  useEffect(() => {
-    const controlNavbar = () => {
-      if (window.scrollY > lastScrollY) {
-        // if scroll down hide the navbar
-        setShow(false);
-      } else {
-        // if scroll up show the navbar
-        setShow(true);
-      }
-
-      // remember current page location to use in the next move
-      setLastScrollY(window.scrollY);
-    };
-    window.addEventListener("scroll", controlNavbar);
-
-    // cleanup function
-    return () => {
-      window.removeEventListener("scroll", controlNavbar);
-    };
-  }, [lastScrollY]);
+  function useShow() {
+    const [Show, setShow] = useState("{}");
```

Comment:
This should be lowercase 

```suggestion
    const [show, setShow] = useState("");
```

Also you'll likely not want curly braces right?